scripts/check_python_docstrings.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export GIT_ROOT=$(readlink -f $(dirname $0)/..)
+
+BASE="$(cd "$(dirname "$(readlink -f "$0")")"; pwd)"
+"$BASE"/check_python.py --pydocstyle "$@"
```

Comment:
We really don't need this wrapper. I just made the other ones for backward compatibly. No reason to have it for new ones.